model/jpa/src/main/java/org/keycloak/authorization/jpa/store/JPAResourceStore.java

```diff
@@ -228,8 +228,12 @@ private void findByOwnerFilter(String ownerId, String resourceServerId, Consumer
                 predicates.add(builder.notEqual(urisSize, 0));
             } else if ("owner".equals(name)) {
                 predicates.add(root.get(name).in(value));
-            } else {
-                predicates.add(builder.like(builder.lower(root.get(name)), "%" + value[0].toLowerCase() + "%"));
+            } else if (!Resource.EXACT_NAME.equals(name)) {
+                if ("name".equals(name) && attributes.containsKey(Resource.EXACT_NAME)) {
```

Comment:
Should we check also the value of `attributes.get(Resource.EXACT_NAME)[0]`? I know at the moment the EXACT_NAME item is only present in case it is true, but this may change in future.

Reply:
I think it is worth adding this, if anything it makes code clearer

Reply:
@pedroigor checking the `value` variable is not what I meant by this comment. In value there is a value of the "name" attribute, however we need to check `attributes.get(Resource.EXACT_NAME)[0]`  which is not in `value` in this for iteration.